docs/UserGuide.md

```diff
@@ -157,11 +178,24 @@ Finds the staff/patients that used to be affiliated or are currently affiliated
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​
 * Adding/Editing a person's affiliation automatically updates the affiliation history for the person.
-* Deleting a person automatically deletes the person from others' affilation history.
 
 Examples:
 * `affnh 2` lists the people who used to be affiliated or are currently affiliated to the 2nd person in the contact list.
 
+### Remove affiliation history of a person: `removeah`
+
+Format: `removeah INDEX`
+
+Remove all affiliation history of patient/staff indicated by the given `INDEX`, except for affiliations that are currently affiliated.
```

Comment:
I haven't seen the first part of the UG but may be good to add a section explaining what's the difference between affiliations and affiliation history (or that we store affiliation history in the first place).

Reply:
Yea I think it is a good idea, we can discuss about it later in class.